pennylane/wires.py

```diff
@@ -106,6 +106,37 @@ def __hash__(self):
         """Implements the hash function."""
         return hash(repr(self.labels))
 
+    def __add__(self, other):
+        """Defines the addition to return a Wires object containing all wires of the two terms.
+
+         **Example:**
```

Comment:
```suggestion
         **Example**
```
😆

With `Args:` and `Kwargs:`, Sphinx now automatically removes the colon in version 2.0+ (in version 1.X, it didn't).

Also, it might be good to move it _below_ the `Args:` and `Returns:`, to match other docstrings

Reply:
Oh yes!